dogpile/cache/backends/redis.py

```diff
@@ -106,6 +112,7 @@ def __init__(self, arguments):
 
         self.lock_timeout = arguments.get("lock_timeout", None)
         self.lock_sleep = arguments.get("lock_sleep", 0.1)
+        self.thread_local_lock = arguments.get("thread_local_lock", True)
```

Comment:
I wonder if we should default to false here?

Reply:
i would need this to be whatever the current default is right now so that I don't have to worry about regressions for existing users.  that said I forgot our conversation about what this flag does (don't worry, that's my job to review, not yours, I just have a lot to get to today) so I don't recall if it was kind of an "always bad" sort of thing...

Reply:
True is the current default.

Reply:
It's only bad/broken for users who want to use redis + distributed locks + async creation runners.